app/jobs/sample_job.rb

```
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SampleJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
```

rubocop (1.68.0)

[warning] Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. If it's unnecessary, remove it. You can also write as `perform(*)` if you want the method to accept any arguments but don't care about them.

(Lint/UnusedMethodArgument)